src/server/routers/pokemon.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const {initPokemonDatabase} = require('../controllers/pokemon')
+
+const router = express.Router();
+
+//will need elevated (owner) role
+
+router.post('/init', initPokemonDatabase)
```

Comment:
Good problem solving skills demonstrated here.

In the real world, you'd probably have some kind of setup script that runs to setup the database... but since we haven't covered that on the course, this is a good workaround.

Be aware that you probably want to protect this route in some way to ensure customers will never be able to access it - and the controller itself should probably check that the db is empty before doing anything, just in case. (Just noticed your `will need elevated (owner) role` comment after writing this, so you've already thought about it. Good work!)

Reply:
Thanks Nathan. 
I would be interested, however,  in seeing a real world example if possible in the future :)